sbws/util/stem.py

```diff
@@ -117,6 +122,33 @@ def _init_controller_socket(socket):
     return c
 
 
+def set_torrc_runtime_options(cont):
+    """Set torrc options at runtime."""
+    try:
+        cont.set_conf('__DisablePredictedCircuits', '1')
+        cont.set_conf('__LeaveStreamsUnattached', '1')
```

Comment:
Can you refactor these options into a dictionary constant?
And let's put all the options in the same place.
You could put the new constant in globals.py after TORRC_STARTING_POINT.

Reply:
That was my initial idea (https://trac.torproject.org/projects/tor/ticket/28692#comment:8):
```
>  Even if only stem.py uses these options, i think they should be in globals.py, since we might need to change them and globals.py should be the place where to change sbws defaults.

Let's leave any refactoring until #28737.
```